KlingonAssistant/data/mem-07-l.xml

```diff
@@ -3322,7 +3322,7 @@ This prefix is sometimes dropped for unknown reasons. While an error, this is of
       <column name="entry_name">lupDujHomwIj lubuy'moH gharghmey.</column>
       <column name="part_of_speech">sen:phr</column>
       <column name="definition">My hovercraft is full of eels.</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Mein Boot is voller Würmer.</column>
```

Comment:
Alright, I didn't know the original source and was translating only the Klingon. In this case, I suggest we take the translation that is found on omniglot: "Mein Luftkissenfahrzeug ist voller Aale."